packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -781,6 +782,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         <div className={ this._classNames.optionsContainerWrapper } ref={ this._resolveRef('_comboBoxMenu') }>
           { (onRenderList as any)({ ...props }, this._onRenderList) }
         </div>
+        { onRenderLowerContent(this.props, this._onRenderLowerContent) }
```

Comment:
I tweaked this to also pass in the default function (in case we ever do return a value!)